src/csharp/Grpc.Tools/build/_grpc/_Grpc.Tools.targets

```diff
@@ -38,6 +38,7 @@
       </Protobuf_Compile>
       <Protobuf_Compile Condition=" '%(Protobuf_Compile.GrpcServices)' == 'Client' ">
         <_GrpcOutputOptions>%(Protobuf_Compile._GrpcOutputOptions);no_server</_GrpcOutputOptions>
+        <_GrpcOutputOptions Condition=" '%(Protobuf_Compile.ClientType)' == 'LiteClient' ">%(Protobuf_Compile._GrpcOutputOptions);lite_client</_GrpcOutputOptions>
```

Comment:
You need to update https://github.com/grpc/grpc/blob/master/src/csharp/Grpc.Tools/build/_grpc/Grpc.CSharp.xml to define a new EnumProperty "ClientType"?